### PR TITLE
F#src 4779 better simulation

### PR DESCRIPTION
**Pull request by ToivoS**

### CODE REVIEW

File: ansible/roles/products/teleop/server/deploy/defaults/main.yml [hunk 1]
```diff
@@ -43,3 +43,6 @@ term_height: 400
 term_x: 200
 term_y: 0
 term_y_correction: 58
+install_nvidia_driver: false
```
Comment (biofotis) [inline ansible/roles/products/teleop/server/deploy/defaults/main.yml:46]: please remove

File: ansible/roles/products/teleop/server/deploy/tasks/main.yml [hunk 2]
```diff
@@ -10,6 +10,11 @@
     server_ip: "{{ groups['server'][0] }}"
   when: groups['server'] is defined
 
+- name: Include installation/nvidia-driver role
```
Comment (biofotis) [inline ansible/roles/products/teleop/server/deploy/tasks/main.yml:13]: please remove

File: README.md [hunk 3]
```diff
@@ -49,9 +49,9 @@ Molecule user guide is available [here](https://molecule.readthedocs.io/en/lates
 
 ## teleop_deploy ##
 
-For deploying teleop software on multiple machines (server, control-machine, client)
+If using real robots, teleop_deploy will deploy software on a laptop (called "server" in this playbook) and a control machine (NUC). If remote_teleop=true, then software will also be deployed on a third computer (called "client").
```
[the remark this answers is not in the document]
Comment (ToivoS) [reply README.md:52]: we do, but only for cyberglove

File: README.md [hunk 4]
```diff
@@ -64,18 +64,24 @@ Open a terminal with Ctrl+Alt+T and run:
...
 
 If no inventory name is provided, and if remote_teleop is not specified or false, then "production" will be automatically selected.
 
 If no inventory name is provided, and if remote_teleop=true, then "production_remote" will be automatically selected.
 
-Example:
+Example for real robots with haptx bimanual teleop:
+
+```bash
+bash <(curl -Ls bit.ly/run-aurora) teleop_deploy --inventory production --read-input docker_username --read-secure docker_password ethercat_interface=enx000ec6bfe185 ethercat_left_hand=enx000ec6c042d5 config_branch=bimanual_demohands_B_D reinstall=true bimanual=true use_aws=true upgrade_check=true image="shadowrobot/teleop-haptx-binary" tag="melodic-v0.0.1" glove=haptx use_steamvr=false arm_ip_right="192.168.1.1" arm_ip_left="192.168.1.2" ethercat_right_arm=eno1 ethercat_left_arm=enx000ec6bfe175
```
[the remark this answers is not in the document]
Comment (ToivoS) [reply README.md:78]: fixed in the last commit

File: ansible/roles/installation/nvidia-driver/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
```
[the remark this answers is not in the document]
Comment (ToivoS) [reply ansible/roles/installation/nvidia-driver/tasks/main.yml:1]: fixed in the last commit

File: README.md [hunk 6]
```diff
@@ -64,18 +64,24 @@ Open a terminal with Ctrl+Alt+T and run:
...
 If no inventory name is provided, and if remote_teleop=true, then "production_remote" will be automatically selected.
 
-Example:
+Example for real robots with haptx bimanual teleop:
+
+```bash
+bash <(curl -Ls bit.ly/run-aurora) teleop_deploy --inventory production --read-input docker_username --read-secure docker_password ethercat_interface=enx000ec6bfe185 ethercat_left_hand=enx000ec6c042d5 config_branch=bimanual_demohands_B_D reinstall=true bimanual=true use_aws=true upgrade_check=true image="shadowrobot/teleop-haptx-binary" tag="melodic-v0.0.1" glove=haptx use_steamvr=false arm_ip_right="192.168.1.1" arm_ip_left="192.168.1.2" ethercat_right_arm=eno1 ethercat_left_arm=enx000ec6bfe175 
+```
+
+Example for simulated robots with a pair of real haptx gloves but without a real vive system:
```
[the remark this answers is not in the document]
Comment (ToivoS) [reply README.md:81]: fixed in the last commit

File: ansible/roles/products/teleop/server/deploy/defaults/main.yml (hunk 1 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/deploy/defaults/main.yml:46]: fixed in the last commit

File: ansible/roles/products/teleop/server/deploy/tasks/main.yml (hunk 2 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/deploy/tasks/main.yml:13]: fixed in the last commit

**Pull request merged**